lblod/frontend-gelinkt-notuleren
# GN-4909: inauguration meeting synchronization support

### Overview
This (draft) PR adds the UI and basic functionality to support the synchronization flow for inauguration meetings.
Specifically, it includes the following features/adjustments:
- Addition of a `synchronizedOn` property on the `installatievergadering` ember-data model. This allows us to check if the inauguration meeting is up-to-date
- Addition of warning concerning data-loss next to each treatment of an agendapoint in the meeting overview
- Addition of a `AuPill` component describing the last synchronization timestamp to inauguration meeting pages. If the user has never synchronized, the pill is not shown.
-  Addition of a configurable `SimpleModal` component. This component extends the `AuModal` component and is less opinionated (you can completely configure it's content).
- Addition of a basic synchronization flow (sync-button + modal + notification)
  * synchronization button which allows the user to open the synchronization modal. The color + text of this button reflects on if the inauguration meeting is up-to-date
  * synchronization modal which displays what will/might be overwritten upon synchronization
  * After successful/failed synchronization, a notification pops up, reflecting whether the sync was (not) successful

TODOs:
- [x] Replace `lorem ipsum` placeholder in sync modal by list of treatments which can be partially overwritten
- [x] Actually sync the different agendapoints in the sync action (requires https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/460)

##### connected issues and PRs:
[GN-4909](https://binnenland.atlassian.net/browse/GN-4909)
https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/460
https://github.com/lblod/app-gelinkt-notuleren/pull/189
https://github.com/lblod/ember-rdfa-editor/pull/1209
https://github.com/lblod/ember-rdfa-editor/pull/1210
https://github.com/lblod/ember-rdfa-editor/pull/1211

### Setup
requires https://github.com/lblod/app-gelinkt-notuleren/pull/189

### How to test/reproduce
- Set-up the backend and start the frontend
- Login as a municipality
- Create an inauguration meeting
- In the header of the meeting page, a button should show up allowing you to synchronize the meeting
- The button should reflect that the meeting is out-of-date (it has never been synchronized yet)
- Open the synchronization modal and synchronize the meeting
- The synchronization modal should now reflect that the meeting is up-to-date
- A pill containing the last synchronization timestamp should now have been added to the header
- When scrolling down, notice the warning pill next to the agendapoint treatments
- Make a data change for the logged in municipality on https://mandatenbeheer.lblod.info
- Notice that (when reloading the meeting), the button now reflects that the meeting is out-of-date
- Make a data change for another municipality on https://mandatenbeheer.lblod.info
- The meeting should still be up-to-date
- If the synchronization succeeds, a notification with a 'success' message appears, this notification disappears automatically
- If the sync fails, a notification with a 'failure' message appears, this notification does not disappear automatically. It gives users tips/instructions on how to solve the issue.

### Checks PR readiness
